assets/stylesheets/bootstrap/_variables.scss

```diff
@@ -334,24 +335,24 @@ $grid-gutter-width:         30px !default;
 //** Point at which the navbar becomes uncollapsed.
 $grid-float-breakpoint:     $screen-sm-min !default;
 //** Point at which the navbar begins collapsing.
-$grid-float-breakpoint-max: ($grid-float-breakpoint - 1) !default;
+$grid-float-breakpoint-max: math.div($grid-float-breakpoint - 1) !default;
```

Comment:
is this correct?
I'm getting this error 
(same thing @ line 351)
```
SassError: Missing argument $number2.
  ┌──> node_modules\bootstrap-sass\assets\stylesheets\bootstrap\_variables.scss
338│ $grid-float-breakpoint-max: math.div($grid-float-breakpoint - 1) !default;
  │                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
  ╵
  ┌──> sass:math
1 │ @function div($number1, $number2) {
  │           ━━━━━━━━━━━━━━━━━━━━━━━ declaration

Reply:
Yep looks wrong. @AprilArcus Can you please fix?

Reply:
Yes, will address this right away.

Reply:
PR: https://github.com/twbs/bootstrap-sass/pull/1225